src/muse/__init__.py

```diff
@@ -1,8 +1,12 @@
 """MUSE model."""
 
 import os
+from importlib.metadata import PackageNotFoundError, version
 
-VERSION = "1.3.0"
+try:
+    __version__ = version("MUSE_OS")
+except PackageNotFoundError:
+    pass
```

Comment:
As mentioned in the python template repo, the following would be best if we are to ignore the exception:
```suggestion
from contextlib import suppress

with suppress(PackageNotFoundError):
    __version__ = version("MUSE_OS")
```
Or giving it a default value.